packages/react-utils/src/components/UtilPageExtra/tests/index.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { ExtraLinks } from '..';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import { HOME_URL } from '../../../constants';
```

Comment:
Kinda weird that we import this from a different place [here](https://github.com/OpenSRP/web/pull/145/files#diff-33465326bb99ab8a1ec9541dd289e4b61ae66cfe0468317a8d6f46fb221fc931R6).

Reply:
I have fixed this.